src/app/api/routes/media.py

```diff
@@ -103,6 +103,7 @@ async def upload_media(media_id: int = Path(..., gt=0),
             status_code=500,
             detail="The upload did not succeed"
         )
+    entry = dict(**entry)
```

Comment:
entry isn't already a dictionary?

Reply:
Nope, it's a record. I got an exception when launching the corresponding test at first.
NB: maybe it's converted implicitly using pydantic when we are in "prod" but the test was failing.